src/components/footer/footer.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC, ReactNode } from 'react'
+import classNames from 'classnames'
+import { NativeProps, withNativeProps } from '../../utils/native-props'
+import { mergeProps } from '../../utils/with-default-props'
+import { Divider } from '../divider/divider'
+import Tag, { TagProps } from '../tag'
+
+const classPrefix = `adm-footer`
+
+export type LinkItem = {
+  label: string
+  value: string
+}
+
+export type ChipItem = {
+  text: string | ReactNode
+  tagProps?: TagProps
+  type?: 'plain' | 'link'
+}
+
+export type FooterProps = {
+  label?: string | ReactNode
+  links?: LinkItem[]
+  content?: string | ReactNode
+  chips?: ChipItem[]
+  onChipClick?: (item: ReactNode | any, index: number) => void
+  onLinkClick?: (item: ReactNode | any, index: number) => void
+} & NativeProps<'--background-color'>
+
+const defaultProps: FooterProps = {
+  label: '',
+  links: [],
+  content: '',
+  chips: [],
+}
+
+export const Footer: FC<FooterProps> = p => {
+  const props = mergeProps(defaultProps, p)
+  const { label, links, content, chips, onChipClick, onLinkClick } = props
+
+  // 点击 标签
+  const clickChipItem = (item: ChipItem, index: number) => {
+    if (chips?.length && item.type === 'link') {
+      onChipClick?.(item, index)
+    }
+  }
+
+  // 点击 链接
+  const clickLinkItem = (
+    item: LinkItem,
+    index: number,
+    e: React.MouseEvent<HTMLAnchorElement, globalThis.MouseEvent>
```

Comment:
```suggestion
    e: React.MouseEvent<HTMLAnchorElement, MouseEvent>
```
这边为什么要用 `globalThis.MouseEvent`？